src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+
+Then, if we look at the top 5 CMSs by rank, we see that WordPress has the highest adoption at for 33.6% of "all" pages:
+
+{{ figure_markup(
+  image="top-cmss-by-rank.png",
+  caption="Top 5 CMSs by rank",
+  description="Bar chart showing the top CMSs percent of share of each ranking category. WordPress had 3.1% of the top 1,000, 8.6% of the top 10,000, 13.2% of the top 100,000, 21.1% of the top 1,000,000, and 33.6% of the top 10,000,000. The remaining CMSs had less than 3% share of all rankings.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=2087254046&format=interactive",
+  sheets_gid="https://docs.google.com/spreadsheets/d/1gAJh4VcSEU6-6aQxBiTFl-7cbyHukfdhg-Iaq9y5pnc/#gid=670045665",
+  sql_file="../cms/top_cms_by_rank.sql"
+  )
+}}
+
+Finally, if we look at the <a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitals Technology Report</a>, we see how each CMS performs by metric:
+
+{{ figure_markup(
+  link="https://datastudio.google.com/s/o6zLzlTpWaI",
+  image="cms_ttfb_cwv_report.png",
+  caption='Origins having good TTFB by CMS (<a hreflang="en" href="https://datastudio.google.com/s/o6zLzlTpWaI">Core Web Vitals Technology Report</a>)',
+  description="Time series chart of Origins Having Good TTFB for WordPress, Wix, and Squarespace. In July, WordPress and Wix were at 5% and Squarespace at 20%."
+  )
+}}
+
+Only 5% of origins on WordPress experienced good TTFB in July 2021. Considering WordPress's large share of the top 10M sites, its poor TTFB could be a contributor to the TTFB degradation by rank.
+
+### First Contentful Paint (FCP)
+
+<a hreflang="en" href="https://web.dev/fcp/">First Contentful Paint (FCP)</a> measures the time from when a load first begins until the browser first renders any contentful part of the page (e.g, text, images, etc.).
+
+{{ figure_markup(
+  image="performance-FCP-by-device.png",
+  caption="FCP performance by device",
+  description="Stacked bar chart showing FCP performance for desktop and mobile devices. For desktop devices, 60% of websites had good (< 1.8s), 27% needs improvement, and 13% poor (>= 3.0s) FCP performance. For mobile, 38% were good, 38% needs improvement, and 24% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1509111466&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+FCP was faster on desktop than mobile, likely due to both faster average network speeds and faster processors. Only 38% of origins had "good" FCP on mobile. Render-blocking resources such as synchronous JavaScript can be a common culprit. Because TTFB is the first part of FCP, poor TTFB will make it difficult to achieve a good FCP.
```

Comment:
Good point. I used quotes when I was referring to the category group directly rather than just describing the results. I could try to make it more consistent or just remove the quotes.